nixos/modules/services/web-servers/phpfpm/default.nix

```diff
@@ -269,8 +299,12 @@ in {
           Type = "notify";
           ExecStart = "${poolOpts.phpPackage}/bin/php-fpm -y ${cfgFile} -c ${iniFile}";
           ExecReload = "${pkgs.coreutils}/bin/kill -USR2 $MAINPID";
-          RuntimeDirectory = "phpfpm";
-          RuntimeDirectoryPreserve = true; # Relevant when multiple processes are running
+          User = poolOpts.user;
+          Group = poolOpts.group;
+          RuntimeDirectory = "phpfpm-${pool}";
+          RuntimeDirectoryMode = "0750";
+          AmbientCapabilities = [ "CAP_CHOWN" ];
+          CapabilityBoundingSet = [ "CAP_CHOWN" ];
```

Comment:
Are these options not required? - `[ "CAP_NET_BIND_SERVICE" "CAP_SETGID" "CAP_SETUID" "CAP_SYS_RESOURCE" ];`

Reply:
- `CAP_NET_BIND_SERVICE` is just for binding on ports below 1024 so not required
- `CAP_SETGID` and `CAP_SETUID` don't seem to be required if `CAP_CHOWN` is provided, but I have only done minimal testing
- `CAP_SYS_RESOURCE` does not sound like it is required, see man page

Reply:
I have discussed with @flokli and this PR actually needs some work yet... more to come.

Reply:
Updated. No longer using `AmbientCapabilities` or `CapabilitiesBoundingSet`.